README.md

```diff
@@ -14,3 +14,11 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+The worst-case run time for the implementation is $\Theta (n^2)$, if the first element of the second subarry is smaller than all elements of the first subarry, this will shift all the element in the first subarry, and merging the two subarrays can take up to $O(n*n)$ time becuase of the element shifting$
```

Comment:
This sounds like you're only analyzing part of the implementation. What's the overall complexity?